apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -213,6 +213,15 @@ export default function GlobalStyle () {
                 }
               }
 
+              .no-content-found-tooltip {
+                .condo-tooltip > .condo-tooltip-content {
+
+                  .condo-tooltip-inner {
+                    width: 600px;
```

Comment:
why 600?

Reply:
removed this style (it was for tooltip that is now replaced by alert text)